src/components/MainComponent/MainComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styles from './MainComponent.module.scss';
+import classNames from 'classnames/bind';
+import RepoList from '../RepoList';
+import { Repo } from '../../modules/repo';
+
+interface Props {
+  repos: Repo[];
+  username: string;
+  onChangeInput: React.ReactEventHandler<HTMLInputElement>;
+  onSearch: React.ReactEventHandler<HTMLFormElement>;
+}
+
+const cx = classNames.bind(styles);
+
+const MainComponent: React.FC<Props> = ({ repos, username, onChangeInput, onSearch}) => {
```

Comment:
```suggestion
const MainComponent: React.FC<Props> = ({ repos, username, onChangeInput, onSearch }) => {
```

eslint 등을 통해 코드 style이 일관되면 더욱 좋을 것 같아요!

Reply:
오 저도 이제까지 협업이나 코드리뷰 해본적이 없어서 제 코드스타일이 쫌 자유분방한데 eslint나 코드프리터좀 적용해봐야겠습니다 감사합니다!